mobile_base2:
  # Publish all joint states -----------------------------------
  mobile_base_controller2:
#    type: "omni_triangle_controller/OmniDriveController"
    type: base_simple_controller/BaseSimpleController
    wheel1: 'wheel1'
    wheel2: 'wheel2'
    wheel3: 'wheel3'
  # arm1_position_controller:
  #   type: position_controllers/JointPositionController
  #   joint: arm1
  # arm2_position_controller:
  #   type: position_controllers/JointPositionController
  #   joint: arm2
  # arm3_position_controller:
  #   type: position_controllers/JointPositionController
  #   joint: arm3
  # arm4_position_controller:
  #   type: position_controllers/JointPositionController
  #   joint: arm4
  # arm5_position_controller:
  #   type: position_controllers/JointPositionController
  #   joint: arm5
  # arm6_position_controller:
  #   type: position_controllers/JointPositionController
  #   joint: arm6


  

